frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -299,49 +299,59 @@ const plugin: Plugin<ConsumedExtensions> = [
     properties: {
       model: PipelineModel,
       loader: async () =>
-        (await import(
-          './components/pipelines/PipelineDetailsPage' /* webpackChunkName: "pipeline-details" */
-        )).default,
+        (
```

Comment:
Another instance of it being a bit aggressive with the parens spacing.  Otherwise think everything else is a clear improvement.